cli/cli/commands/kurtosis_context/context_switch/switch.go

```diff
@@ -120,11 +123,11 @@ func SwitchContext(
 			}
 		}
 	} else {
-		if store.IsRemote(currentContext) {
-			return stacktrace.NewError("New context is remote but Kurtosis Portal is not reachable locally. " +
-				"Make sure Kurtosis Portal is running before switching to a remote context again.")
+		if err := portalManager.StopExisting(ctx); err != nil {
```

Comment:
Probably fine for now, I'm wondering if we shouldn't live it running at some point. It's pretty lightweight anyway. maybe add a comment mentioning this possibility for future readers of this code